doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -4618,6 +4618,25 @@ The fully expanded example above (without environment variables) looks like this
 
```

Comment:
The docs look great. Can you please add a release note snippet? Please see https://guides.dataverse.org/en/6.3/developers/version-control.html#writing-a-release-note-snippet

Reply:
Has been added in [3f81981](https://github.com/IQSS/dataverse/pull/10739/commits/3f81981287643f2ac7d37bce8f29bc86f9088ca6).